examples/server-sent-events/src/main/java/example/armeria/server/sse/Main.java

```diff
@@ -52,17 +49,17 @@ private static Server newServer(int httpPort, int httpsPort, Duration sendingInt
     static void configureServices(ServerBuilder sb, Duration sendingInterval, long eventCount,
                                   Supplier<String> randomStringSupplier) throws Exception {
         sb.service("/long", (ctx, req) -> {
-            // Note that you MUST adjust the request timeout if you want to send events for a
-            // longer period than the configured request timeout. The timeout can be disabled by
-            // 'clearRequestTimeout()' like the below, but it is NOT RECOMMENDED in
-            // the real world application, because it can leave a lot of unfinished requests.
-            ctx.clearRequestTimeout();
-            return ServerSentEvents.fromPublisher(
-                    Flux.interval(sendingInterval)
-                        .onBackpressureDrop()
-                        .take(eventCount)
-                        .map(unused -> ServerSentEvent.ofData(randomStringSupplier.get())));
-        }).annotatedService(new Object() {
+              // Note that you MUST adjust the request timeout if you want to send events for a
+              // longer period than the configured request timeout. The timeout can be disabled by
+              // 'clearRequestTimeout()' like the below, but it is NOT RECOMMENDED in
+              // the real world application, because it can leave a lot of unfinished requests.
+              ctx.clearRequestTimeout();
+              return ServerSentEvents.fromPublisher(
+                      Flux.interval(sendingInterval)
+                          .onBackpressureDrop()
+                          .take(eventCount)
+                          .map(unused -> ServerSentEvent.ofData(randomStringSupplier.get())));
+          }).annotatedService(new Object() {
```

Comment:
Incorrect indentation?

Reply:
It was reformatted in my IntelliJ and LINE OSS schema is set 🤔

Reply:
I also have checked it and the code format was wrong. 😄